lms/static/scripts/frontend_apps/components/Dialog.js

```diff
@@ -63,7 +63,7 @@ export default function Dialog({
       aria-labelledby={dialogTitleId}
       aria-modal="true"
       ref={rootEl}
-      tabIndex="-1"
+      tabIndex={-1}
```

Comment:
Passing a string for `tabIndex` happens to work due to implicit coercions, but the `tabIndex` property of DOM elements is an integer.

Reply:
good to know!